package.json

```diff
@@ -26,14 +26,20 @@
     "@nestjs/core": "^7.0.0",
     "@nestjs/mongoose": "^7.0.2",
     "@nestjs/platform-express": "^7.0.0",
-    "@nestjs/swagger": "^4.7.2",
+    "@nestjs/swagger": "^4.8.2",
+    "@nestjs/typeorm": "^7.0.0",
     "@types/convert-excel-to-json": "^1.7.1",
     "convert-excel-to-json": "^1.7.0",
```

Comment:
Si es posible, limpiar/reducir las dependencias asociadas a xls/csv/....
- @types/convert-excel-to-json
- convert-excel-to-json
- read-excel-file
- xlsx

Reply:
Se elimina solo read-excel-file ya que xlsx es la usada en este momento y convert-excel-to-json es usada para lo que estaba antes